### PR TITLE
docs(WEBRTC-234): add typedoc on setVideoSettings method

### PR DESCRIPTION
**Pull request by DeividVeloso**
- add typedoc on setVideoSettings method

## 📝 To Do

- [x] All linters pass
- [ ] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. Navigate into `package/js/examples/react-audio`
2. Click on `Set Video Settings` button
3. See if returns an object with the default mic label and with the applied settings.

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [ ] Firefox
- [ ] Safari

## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |image|

### CODE REVIEW

File: packages/js/src/Modules/Verto/BrowserSession.ts [hunk 1]
```diff
@@ -445,8 +445,43 @@ export default abstract class BrowserSession extends BaseSession {
     return { audio: this._audioConstraints, video: this._videoConstraints };
   }
 
+  /**
+   * setAudioSettings
+   *
+   * You can set the default `audio` constraints for your client. [See here](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#Properties_of_audio_tracks) for further details.
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/BrowserSession.ts:451]:
```suggestion
   * Sets the default `audio` constraints for your client. [See here](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#Properties_of_audio_tracks) for further details.
```

Comment (hugobdev) [reply packages/js/src/Modules/Verto/BrowserSession.ts:451]: Again, we should keep docs in the infinitive.

File: packages/js/src/Modules/Verto/BrowserSession.ts [hunk 2]
```diff
@@ -509,8 +544,44 @@ export default abstract class BrowserSession extends BaseSession {
     this._audioConstraints = true;
   }
 
+  /**
+   * setVideoSettings
+   *
+   * You can set the default `video` constraints for your client. [See here](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#Properties_of_video_tracks) for further details.
```
Comment (hugobdev) [inline packages/js/src/Modules/Verto/BrowserSession.ts:550]:
```suggestion
   * Sets the default `video` constraints for your client. [See here](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints#Properties_of_video_tracks) for further details.
```

**Review by hugobdev — CHANGES_REQUESTED** [review]

**Review by SuaYoo — COMMENTED** [review]
Did you mean to branch this from https://github.com/team-telnyx/webrtc/pull/73 ? Looks like there is duplicate changes

**Comment by DeividVeloso** [comment]
> Did you mean to branch this from #73 ? Looks like there is duplicate changes

I cloned from #73, because the code is pretty equal, we need to merge 73 first.

**Comment by SuaYoo** [comment]
> > Did you mean to branch this from #73 ? Looks like there is duplicate changes
> 
> I cloned from #73, because the code is pretty equal, we need to merge 73 first.

I see your changes in #73 in this PR when I look at the files changed. I tried changing the base, but it looks like you may have accidentally copied `setAudioSettings` altogether and will need to delete those lines from this PR.

File: packages/js/examples/react-audio/stories/components/Utilities.js [hunk 3]
```diff
@@ -108,6 +108,65 @@ function Utilities({ environment, username, password }) {
...
+          Returns the audio settings applied for <b>{label}</b>
+        </span>
+      ),
+      message: (
+        <pre style={{ display: 'block', backgroundColor: '#ccc' }}>
+          {JSON.stringify(results, undefined, 2)}
+        </pre>
+      ),
+    });
+  };
```
Comment (SuaYoo) [inline packages/js/examples/react-audio/stories/components/Utilities.js:138]: This is the duplicated code from https://github.com/team-telnyx/webrtc/pull/73/files

File: packages/js/src/Modules/Verto/BrowserSession.ts [hunk 4]
```diff
@@ -445,8 +445,41 @@ export default abstract class BrowserSession extends BaseSession {
...
+   *
+   * ```js
+   * // within an async function
+   * const constraints = await client.setAudioSettings({
+   *  micId: '772e94959e12e589b1cc71133d32edf543d3315cfd1d0a4076a60601d4ff4df8',
+   *  micLabel: 'Internal Microphone (Built-in)',
+   *  echoCancellation: false
+   * })
+   * ```
+   */
```
Comment (SuaYoo) [inline packages/js/src/Modules/Verto/BrowserSession.ts:475]: This as well: https://github.com/team-telnyx/webrtc/pull/73/files#diff-2447a25963f6b87422abbb0c3432ebcd9ebaf2c4e9abef301057d962b60a45b6

**Review by hugobdev — APPROVED** [review]

**Pull request merged**